src/parser.py

```diff
@@ -89,15 +103,52 @@ def create_contracheque(row):
     for i in range(len(items)):
         key, value = items[i][0], items[i][1]
         remuneration = Coleta.Remuneracao()
-        remuneration.natureza = Coleta.Remuneracao.Natureza.Value("R")
         remuneration.categoria = OBRIGATORIOS
         remuneration.item = key
-        remuneration.valor = float(number.format_value(row[value]))
+        if month == "12" and value in [13, 14] and employee != "":
+            for emp in employee.remuneracao:
+                if key in emp.item:
+                    remuneration.valor = float(number.format_value(row[value])) + emp.valor
+        else: 
+            remuneration.valor = float(number.format_value(row[value]))
         remuneration.natureza = Coleta.Remuneracao.Natureza.Value("D")
         remuneration_array.remuneracao.append(remuneration)
 
     return remuneration_array
 
+def contracheque13(cq13):
+    dict_cq13 = {}
+    for row in cq13:
+        if not number.is_nan(row[0]):
+            mat = str(row[0])[:-2]
+            remuneracoes = dict_cq13.get(mat, Coleta.Remuneracoes())
+            rem = Coleta.Remuneracao()
+            items = list(MES13[EVENTUALTEMP].items())
+            for i in range(len(items)):
```

Comment:
Creio que você pode usar a iteração de dicionários em python e, dessa forma, simplificar esse código (ficando mais legível)

https://www.horadecodar.com.br/2020/07/08/como-iterar-em-dicionarios-utilizando-for-em-python/